### DISCUSSION TITLE
Admin Page Load Event

### DISCUSSION THREAD

**Discussion by magnessjo** [craftcms/ckeditor#70] [Ideas]
Please let me know if this question should be moved to the Craftcms repo.  It isn't specific to the CKEditor plugin, but I found it while working on the plugin, so I entered it here. 

When trying to initialize a custom build for CKEditor, I noticed that the code was being executed before the textarea fields were created (even on the window load event).  I assume that Craft and a plugin (Neo) are doing a bunch of JS work as well on load to layout the page.  I did [hacky things](https://github.com/UMD-Digital/ckeditor-builder/blob/master/source/index.ts#L35) to make it work.  I could probably figure a semi better implementation, but it would still be fragile. Are there any better solutions than iterating over the page?  

Note: I attempted to load the JS via the asset bundle (I thought maybe that would load last - No idea why it would, but I tried it).

**Comment by brandonkelly** [comment]
Just added a new `craft\ckeditor\Field::EVENT_MODIFY_CONFIG` event for the upcoming 3.1.0 release, which allows you to completely customize the CKEditor configuration, before gets passed to `ClassicEditor.create()`. (612c1c5d78d04342d221f2e66a29a4d155218951)

**Reply by brandonkelly** [reply]
3.1.0 is now tagged with that event 🎉
